Include/internal/pycore_typeobject.h

```diff
@@ -148,6 +148,9 @@ typedef int (*_py_validate_type)(PyTypeObject *);
 extern int _PyType_Validate(PyTypeObject *ty, _py_validate_type validate, unsigned int *tp_version);
 extern int _PyType_CacheGetItemForSpecialization(PyHeapTypeObject *ht, PyObject *descriptor, uint32_t tp_version);
 
+// Precalculates count of non-unique slots and fills wrapperbase::name_count.
```

Comment:
```suggestion
// Precalculates count of non-unique slots and fills wrapperbase.name_count.
```

`::` is primarily used in C++ which would be confusing here.